grails-app/services/au/org/ala/biocache/hubs/UserService.groovy

```diff
@@ -25,6 +25,12 @@ class UserService {
     @Value('${dataquality.enabled}')
     boolean dataQualityEnabled
 
+    @Value('${dataquality.disableAllFilters:false}')
```

Comment:
I'm wondering if this var can be clearer, as `dataquality.disableAllFilters` seems to contradict `dataquality.enabled` (to a new developer looking at code for first time). How about `dataquality.disableFiltersByDefault` ?

Reply:
Yes, I'll make the change. Actually for ansible, I used the word default.